src/Shared/Model/Mutators/SwapOrderOfLettersMutator.cs

```diff
@@ -0,0 +1,30 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// --------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.CST.OpenSource.Model.Mutators
+{
+    /// <summary>
+    /// Generates mutations for if a letter was swapped with an adjacent letter.
+    /// </summary>
+    public class SwapOrderOfLettersMutator : BaseMutator
+    {
+        public new string Mutator = "LETTERS_SWAPPED";
+
+        public override IEnumerable<(string Name, string Reason)> Generate(string arg)
+        {
+            if (arg.Length < 5)
+            {
+                // TODO: Figure out why we only check for less than 5 char strings.
```

Comment:
I'd be interested in learning what is restricting us to five or more character names

Reply:
Combinatorial complexity is my guess. By default this hits the real production servers to query packages.

Reply:
Took a closer look at this function and its a O(N) loop, I dont see any reason to keep the <5.